R/cal-plot.R

```diff
@@ -408,6 +408,10 @@ binary_plot_impl <- function(tbl, x, y,
     }
     has_groups <- TRUE
     dplyr_group <- parse_expr(gp_vars)
+    grouping_var <- tbl[, gp_vars][[1]]
+    if(is.numeric(grouping_var)) {
+      tbl[, gp_vars] <- as.factor(grouping_var)
```

Comment:
```suggestion
      tbl[, gp_vars] <- as.factor(format(grouping_var))
```

Reply:
Do we allow 2+ grouping variables? Should this use `across()` instead?

Reply:
At this point it only supports one grouping variable: https://github.com/tidymodels/probably/blob/407d6b5d167d2d65f24f23e033aad865e0d3ed5a/R/cal-plot.R#L407